satellite/satellitedb/migrate.go

```diff
@@ -736,6 +736,22 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					`ALTER TABLE offers ALTER COLUMN expires_at SET NOT NULL;`,
 				},
 			},
+			{
+				Description: "Create value attribution table",
+				Version:     27,
+				Action: migrate.SQL{
+					`CREATE TABLE IF NOT EXISTS value_attributions (
```

Comment:
we are avoiding using `IF NOT EXISTS` because if it exists then it indicates a bigger problem with migration and we can miss it.